app/fullService/axiosFullService.ts

```diff
@@ -62,9 +62,10 @@ const axiosFullService = async <T>(
       throw new Error(errorMessage);
     }
 
-    // TODO: determine if we need to handle errors here or elsewhere
-    // such as the API or services
-    return camelCaseObjectKeys(response);
+    return {
+      error: response.error ? response.error.data?.details ?? 'unknown full-service error' : null,
```

Comment:
this line has 5 statements in it, making it a... quinternary? This logic should probably be pulled out for readability.